src/Symfony/Component/Form/Tests/AbstractLayoutTest.php

```diff
@@ -62,7 +62,7 @@ protected function assertMatchesXpath($html, $expression, $count = 1)
         try {
             // Wrap in <root> node so we can load HTML with multiple tags at
             // the top level
-            $dom->loadXML('<root>'.$html.'</root>');
+            $dom->loadHTML('<!DOCTYPE html><html><body>'.$html.'</body></html>');
```

Comment:
using `loadHTML` enables HTML entities (`Entity 'euro' not defined` otherwise). It makes sense; we're in fact loading $html.